### PR TITLE
feat(wren-ui): add summary to thread response

### PR DESCRIPTION
**Pull request by wwwy3y3**
## Description
* Add `summary` to thread response. It affects following APIs
  ```
  thread(threadId: Int!): DetailedThread!
  threadResponse(responseId: Int!): ThreadResponse!
  ```

  Screenshot 2024-04-25 at 6 38 16 PM
  > Add a summary field to  `ThreadResponse`

* Deprecate `sql` field in `Thread` & `DetailedThread` at our graphql API
  Screenshot 2024-04-25 at 6 34 21 PM

### CODE REVIEW

File: wren-ui/migrations/20240425000000_add_thread_response_summary.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+// add pg related columns to project table
```
Comment (andreashimin) [inline wren-ui/migrations/20240425000000_add_thread_response_summary.js:5]: The note is not relevant to the migration

File: wren-ui/migrations/20240425000000_add_thread_response_summary.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+exports.up = function (knex) {
+  return knex.schema.alterTable('thread_response', (table) => {
+    // pg
```
Comment (andreashimin) [inline wren-ui/migrations/20240425000000_add_thread_response_summary.js:8]:
```suggestion
```

**Review by onlyjackfrost — APPROVED** [review]

File: wren-ui/migrations/20240425000000_add_thread_response_summary.js (hunk 2 above)
Comment (wwwy3y3) [reply wren-ui/migrations/20240425000000_add_thread_response_summary.js]: addressed in ce35867

**Review by andreashimin — APPROVED** [review]
LGTM

**Pull request merged**